.gitignore

```diff
@@ -49,3 +49,7 @@ docs/.cache/
 *.key
 
 .cache/
+
+# test
+.pytest_cache
+.coverage
```

Comment:
fyi, test ignored files under `$FOGLAMP_ROOT/tests/.gitignore` and I think this is the right place to maintain it. 
We have to run pytest command under `$FOGLAMP_ROOT/tests/unit/python/foglamp`, though pytest autodiscover the tests but right now integration tests are creating a problem. If we want to run all the tests outside the tests dir path then do `pytest --ignore=tests/integration`